examples/Gameraww/app/fetch.js

```diff
@@ -0,0 +1,15 @@
+module.exports = function fetch(path) {
```

Comment:
It's easier for me to develop and test out new things in Gameraww :grin: I suppose that as soon as modules are mature enough we can move this to ES6 as well.